### DISCUSSION TITLE
Sort Alphabetically Automatically

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1563] [Q&A] [answered]
Is it possible to have all the notes sorted alphabetically automatically?
At the moment, after I create a new note, I select the root and press ALT + S. It would be handy if this could be handled automatically.

**Comment by zadam** [comment, marked as answer]
You can add label `sorted` to a note, then its children will get sorted automatically. If you make this label inheritable then this will be applied to the whole subtree.

**Reply by hokutonanto** [reply]
Thank you, @zadam 

This is what I added to the root
![image](https://user-images.githubusercontent.com/23059969/105554130-95650f00-5cbb-11eb-88b0-69dad3042d37.png)

Is that correct?
Now, when I create a new child node it also shows this
![image](https://user-images.githubusercontent.com/23059969/105554180-ab72cf80-5cbb-11eb-8703-33caa4f6e523.png)

But it doesn't appear ordered alphabetically.
I guess I'm doing something wrong?

**Reply by zadam** [reply]
Hi, you defined the label correctly.

I checked this and found out that inherited `sorted` labels were not actually supported. I fixed that in stable branch which will be released in 0.45.9. Once you update to this new version, you will have to define this label again so that the resorting is activated.

Thanks for bringing up this problem to my attenation!

**Reply by hokutonanto** [reply]
Awesome. Thanks!
